conversions/infix_to_postfix.c

```diff
@@ -0,0 +1,223 @@
+/**
+ * @file
+ * @brief Infix to Postfix Expression Conversion
+ * @details Convert Infixed expressions to Postfix expression.
+ * @author [Harsh Karande](https://github.com/harshcut)
+ * @see [notes](https://condor.depaul.edu/ichu/csc415/notes/notes9/Infix.htm)
+ */
+
+// include header files
+#include <stdio.h>  /// for printf() and scanf()
+
+// globally declared structure
```

Comment:
Provide a brief description of what the structure does or what is it for.
```suggestion
/**
 * @brief description of the structure
 */
```